src/main/scripts/jbang

```diff
@@ -219,8 +219,10 @@ fi
 
 ## run it using command substitution to have just the user process once jbang is done
 export JBANG_RUNTIME_SHELL=bash
-export JBANG_STDIN_NOTTY=$([ -t 0 ] && echo "false" || echo "true")
+JBANG_STDIN_NOTTY=$([ -t 0 ] && echo "false" || echo "true")
```

Comment:
why did this line get split (and the other not)?

Reply:
See https://www.shellcheck.net/wiki/SC2155 ... so it's because this line uses `$( ... )` and the other two do not.